package.json

```diff
@@ -129,7 +129,7 @@
     "redux-mock-store": "1.5.3",
     "snyk": "^1.161.1",
     "source-map-explorer": "1.8.0",
-    "typescript": "3.5.3"
+    "typescript": "3.7.5"
   },
```

Comment:
Why we need to upgrade this ? In https://github.com/kiali/kiali-ui/pull/1654 the test passed correctly

Reply:
Yeah, but this PR is exactly to upgrade Typescript :)